industrial_ci/src/docker.sh

```diff
@@ -65,8 +65,8 @@ function ici_require_run_in_docker() {
 #######################################
 function ici_run_cmd_in_docker() {
   local run_opts=($DOCKER_RUN_OPTS)
-  local commit_image=$_COMMIT_IMAGE
-  unset _COMMIT_IMAGE
+  local commit_image=$COMMIT_IMAGE
```

Comment:
If the role of `COMMIT_IMAGE` is nothing more than boolean, then we can check by other `COMMIT_IMAGE`-related variables whether to enable this feature or not. This way we can omit COMMIT_IMAGE variable.